### ISSUE TITLE
Converter

### ISSUE THREAD

**Issue by MertCokelek** [MertCokelek/VisualSaliencyTransformer#2]
Transformer
- Transformer Encoder
- Block (transformer Block)
- nn.LayerNorm

TokenTransformer
- nn.LayerNorm
- nn.Linear
- nn.GeLU
- token_transformerEncoder
  - asd
    - Block
    - nn.LayerNorm
  - saliency - contour_token_inference
    - nn.Linear
    - nn.Dropout
    - nn.Sigmoid
